src/SourceBuild/content/build.proj

```diff
@@ -20,5 +20,6 @@
   </Target>
 
   <Import Project="$(RepositoryEngineeringDir)build.sourcebuild.targets" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)build.targets" />
```

Comment:
Please make the targets file name more descriptive, i.e. `sdkarchivediff.targets`.